SU-HSDO/suhumsci
# HSD8-666 Provision gavin-wright.humsci.stanford.edu

# READY FOR REVIEW

# Summary
- Provision new site: gavin-wright.humsci.stanford.edu

# Needed By (Date)
- asap

# Urgency
- high (gotta get it in this release)

# Steps to Test
1. review code
1. optionally checkout this branch
1. ensure `drush @gavin_wright__humsci.local st` responds with applicable information.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
